context/app/static/js/components/organ/Assays/Assays.jsx

```diff
@@ -17,9 +17,11 @@ import { useDatasetTypeMap } from 'js/components/home/HuBMAPDatasetsChart/hooks'
 import { Flex, StyledInfoIcon, StyledDatasetIcon } from '../style';
 import { getSearchURL } from '../utils';
 
-function Assays({ organTerms }) {
+function Assays({ organTerms, onHandleSearchStatus }) {
   const assayTypeMap = useDatasetTypeMap();
 
+  const [bucketData, setBucketData] = useState([]);
```

Comment:
Not sure why we would need to add this to state? We can continue to access the buckets as we have.